config/manager/manager.yaml

```diff
@@ -17,7 +17,7 @@ spec:
   selector:
     matchLabels:
       control-plane: controller-manager
-  replicas: 1
+  replicas: 2
```

Comment:
This is ASOv1's config, you want to change ASOv2's, which is [here](https://github.com/Azure/azure-service-operator/tree/main/v2/config/manager).

Same goes from the `config/default/manager_pod_disruption_budget.yaml` file above - it should be in `v2/config/manager` instead I think.

Reply:
Thanks, this was moved now.